template_profiler_panel/templates/template_profiler_panel/template.html

```diff
@@ -25,8 +25,8 @@ <h4>{{ templates|length }} {% trans 'calls to Template.render()' %}</h4>
         <td>{{ template.name }}</td>
         <td class="timeline">
           <div class="djDebugTimeline">
-            <div class="djDebugLineChart" style="left: {{ template.offset_p|stringformat:'f' }}%;">
-              <strong title="Start {{ template.start }}" style="min-width: 1px; width: {{ template.rel_duration_p|stringformat:'f' }}%; background-color: {{ template.color.bg }};">&nbsp;</strong>
+            <div class="djDebugLineChart" style="margin-left: {{ template.offset_p|stringformat:'f' }}%;">
+              <div title="Start {{ template.start }}" style="min-width: 1px; width: {{ template.rel_duration_p|stringformat:'f' }}%; background-color: {{ template.color.bg }};">&nbsp;</div>
```

Comment:
why the change from `strong`?

Reply:
Because on `strong` tag the parameter `width` has no effect in Chrome.

Reply:
See this:
![Snímek z 2021-01-12 16-59-52](https://user-images.githubusercontent.com/156755/104339252-af526500-54f7-11eb-93ec-abe755600a9a.png)
![Snímek z 2021-01-12 16-59-40](https://user-images.githubusercontent.com/156755/104339249-aeb9ce80-54f7-11eb-9187-ae6194105b59.png)

Reply:
Now I tried also in Firefox, and the effect is the same.